torchinductor/codegen/triton.py

```diff
@@ -934,11 +934,7 @@ def codegen_kernel(self, name=None):
 
         argdefs, _ = self.args.python_argdefs()
 
-        if config.dynamic_shapes:
-            maybe_const = ""
-        else:
-            maybe_const = ": tl.constexpr"
-
+        maybe_const = ""
```

Comment:
its only used on L940, you can probably just rm the whole thing. 

Or leave it commented out with an explanation?

Reply:
Yeah I didn't remove it because we might want to go back to it, I'll leave a comment.

Reply:
We also have `argdefs.append(f"{tree.prefix.upper()}BLOCK : tl.constexpr")` - is that one alright?

Reply:
Yes those need to be constexprs and powers of 2 - triton codegen wouldn't work otherwise.